dts/bindings/can/st,stm32-can.yaml

```diff
@@ -13,3 +13,8 @@ properties:
 
     clocks:
       required: true
+
+    master-can-reg:
```

Comment:
I'm ok with this, but can we open an issue to fix this in the future to just be a phandle to the master node.

Reply:
#24686